pkgs/racket-doc/scribblings/reference/pairs.scrbl

```diff
@@ -1398,6 +1398,52 @@ See also @racket[max].
   (argmax car '((3 pears) (1 banana) (2 apples)))
   (argmax car '((3 pears) (3 oranges)))]}
 
+
+@defproc[(best+key+index [lst (and/c pair? list?)]
+                         [better? (-> any/c any/c any/c)]
+                         [#:key key (-> any/c any/c) values])
+         (values any/c any/c exact-nonnegative-integer?)]{
+Returns the element @racket[x] of @racket[lst], the value @racket[k]=@racket[(key x)]
+and the position of @racket[x] in @racket[lst] such that @racket[k] is the best
```

Comment:
"the first best"?

Reply:
It can be the last best if you use `<=` instead of `<`.

Reply:
Ah! Nice trick ;)

Reply:
Do you want to add a paragraph to spell out this trick explicitly?

Reply:
An example that shows the different behavior of `<` and `<=` would be nice as well.

Reply:
It's already explained on line 1436, but maybe it would be better earlier, and indeed an example is required!

Reply:
Ah, the examples are actually on lines 1415 and 1417, but they are far away from the explanation. I'll regroup.

Reply:
Ohhh, I forgot that `best+key+index` returns the index as well, so what you already have is suffice for my request!

Since `best+key+index` appears before `best`/`best-key`/`best-index`, I hope that the description about "Note that @racket[(best lst < #:key key)] is equivalent to @racket[(argmin lst key)], but @racket[(best lst <= #:key key)] returns instead the @emph{last} smallest element of @racket[lst]." will be moved here.

Reply:
Good point, done :)

Reply:
Done (as in, actually pushed)